openzaak-alfresco-share/src/main/resources/alfresco/web-extension/messages/openzaak-alfresco-share.properties

```diff
@@ -0,0 +1,95 @@
+# Type labels
+type.drc_zaaktypefolder=Openzaak zaaktype folder
+type.drc_zaakfolder=Openzaak zaakfolder
+
+type.drc_document=Openzaak document
+type.drc_gebruiksrechten=Openzaak gebruiksrechten
```

Comment:
this is rather `Documenten API gebruiksrechten`, this applies for everything of `type.drc_`

Could maybe also be named "Open Zaak Documenten API gebruiksrechten" but that's quite verbose.

Reply:
I will be using the shorter version. The window those labels are shown in is not that large